container-core/src/main/java/com/yahoo/jdisc/http/server/jetty/HttpRequestDispatch.java

```diff
@@ -119,6 +120,10 @@ public void dispatch() throws IOException {
                             error,
                             () -> "Network connection was unexpectedly terminated: " + parent.jettyRequest.getRequestURI());
                     parent.metricReporter.prematurelyClosed();
+                } else if (isErrorOfType(error, TimeoutException.class)) {
```

Comment:
Just jetty that sende these here?

Reply:
Exceptions thrown by com.yahoo.jdisc.handler.RequestHandler will be treated equally with Jetty's exceptions